src/components/structures/RoomView.js

```diff
@@ -572,6 +588,15 @@ module.exports = React.createClass({
         });
     },
 
+    _onSearchResultSelected: function(searchResult) {
+        this.setState({searchContext: searchResult.context});
+    },
+
+    getSearchContextTiles: function() {
+        return this._timelineToTiles(
+            this.state.searchContext.getTimeline(),
+            !!this.state.searchContext.startToken);
```

Comment:
Can we not bang bang cast please? `Boolean(this.state.searchContext.startToken)` is A Thing and is a lot clearer.